.github/workflows/update-timetable.yml

```
@@ -0,0 +1,43 @@
+      with:
+        python-version: '3.8'
+    
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r scripts/requirements.txt
+    
+    - name: Create Google Auth credentials
+      run: |
+        echo "${{ secrets.GOOGLE_CREDENTIALS }}" > credentials.json
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)